frontend/app/services/personal-information-service.server.ts

```diff
@@ -151,41 +246,44 @@ function createPersonalInformationService() {
   }
 
   function toPersonalInformation(personalInformationApiResponse: PersonalInformationApiResponse): PersonalInfo {
-    const addressList = personalInformationApiResponse.Client.PersonContactInformation?.Address;
+    const addressList = personalInformationApiResponse.BenefitApplication.Applicant?.PersonContactInformation?.Address;
     const homeAddressList = addressList?.filter((address) => address.AddressCategoryCode.ReferenceDataName == 'Home');
     const mailingAddressList = addressList?.filter((address) => address.AddressCategoryCode.ReferenceDataName == 'Mailing');
 
     return {
-      applictantId: personalInformationApiResponse.Client.ClientIdentification ? personalInformationApiResponse.Client.ClientIdentification.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText === 'Applicant ID').at(0)?.IdentificationID : '',
-      clientId: personalInformationApiResponse.Client.ClientIdentification?.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText === 'Client Number').at(0)?.IdentificationID,
-      firstName: personalInformationApiResponse.Client.PersonName?.at(0)?.PersonGivenName?.at(0),
-      lastName: personalInformationApiResponse.Client.PersonName?.at(0)?.PersonSurName?.at(0),
-      emailAddress: personalInformationApiResponse.Client.PersonContactInformation?.EmailAddress.EmailAddressID,
-      phoneNumber: personalInformationApiResponse.Client.PersonContactInformation?.TelephoneNumber.FullTelephoneNumber,
+      applicantCategoryCode: personalInformationApiResponse.BenefitApplication.Applicant?.ApplicantCategoryCode.ReferenceDataID,
+      applictantId: personalInformationApiResponse.BenefitApplication.Applicant?.ClientIdentification?.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText === 'Applicant ID').at(0)?.IdentificationID,
+      clientId: personalInformationApiResponse.BenefitApplication.Applicant?.ClientIdentification?.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText === 'Client ID').at(0)?.IdentificationID,
+      clientNumber: personalInformationApiResponse.BenefitApplication.Applicant?.ClientIdentification?.filter((clientInfoDto) => clientInfoDto.IdentificationCategoryText == 'Client Number').at(0)?.IdentificationID,
```

Comment:
I noticed that the newly introduced `clientNumber` field maps to the `'Client Number'`, but previously value was mapped to `clientId`. We definitely need a field that represents the the `'Client Number'` from the response, but we should remove the other field (the one that is mapped using the `'Client ID'` field in the response) as it won't be used by the application. 

The value of `'Client Number'` will be used as the `clientId` query to call CCT to get all letters: https://github.com/DTS-STN/canadian-dental-care-plan/blob/c3bf1024250f1e71383b4b01159a2ba95bbfcc9d/frontend/app/services/letters-service.server.ts#L86

Reply:
I believe Guillaume mentioned we are using it for something else and to keep the mapping for now.

Reply:
I'm okay with leaving both fields if you feel there's a need, but we just have to update the `getLetters(..)` call to pass in the new `clientNumber` field (the `clientId` is currently passed, which would be incorrect with our mapping change in this PR).